.github/workflows/build-and-test.yml

```
@@ -36,3 +36,6 @@ jobs:
 
       - name: Build
         run: pnpm build
+
+      - name: Build App with Storybook 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)